### PR TITLE
Bob/update metabase view permissions

### PR DESCRIPTION
**Pull request by fzhao99**
# DATABASE PULL REQUEST

Adds the is_deleted column to the org queue table view in Metabase

## Related Issue

Adding this column to get insight into how often duplicate orgs were deleted over the course of time.

## Changes Proposed

Add in relevant Liquibase changes to expose that column to the Metabase viewer

## Testing

- How should reviewers verify this PR?

## Checklist for Primary Reviewer

- [x] Only database changes are included in this PR
- [x] Any new tables or columns that do not contain PII are accompanied by a GRANT SELECT to the no-PHI user
- [x] Any changes to tables that have custom no-PHI views are accompanied by changes to those views (including re-granting permission to the no-PHI user if need be)
- [x] Each new changeset has a corresponding [tag](https://docs.liquibase.com/change-types/community/tag-database.html)
- [ ] Rollback has been verifed locally and in a deployed environment

### CODE REVIEW

File: backend/src/main/resources/db/changelog/db.changelog-master.yaml [hunk 1]
```diff
@@ -3916,5 +3916,23 @@ databaseChangeLog:
...
\ No newline at end of file
+        - sql: 
+            GRANT SELECT ON TABLE ${database.defaultSchemaName}.api_audit_event_no_phi TO ${noPhiUsername};
+  - changeSet:
+      id: update-no-phi-user-view-of-org-queue-table
+      author: bob@skylight.digital
+      comment: Add the organization_queue's is_deleted column to the view permissions of the no-phi-user so we can see it in metabase
+      changes:
+         - tagDatabase:
+            tag: update-org-queue-view
```
Comment (emmastephenson) [inline backend/src/main/resources/db/changelog/db.changelog-master.yaml:3927]: We try to keep the id and tag identical for easier rollbacks, could you change the tag to `update-no-phi-user-view-of-org-queue-table`?

**Comment by emmastephenson** [comment]
@emyl3 and @johanna-skylight, CC'ing you both for PR visibility here - this is a database change to allow one of our tables to get a fuller view in Metabase. Feel free to DM me if you have any questions about it.

**Review by emmastephenson — DISMISSED** [review]
Looks great, thanks Bob!

**Review by emmastephenson — APPROVED** [review]

**Review by nickclyde — APPROVED** [review]
Looks great!

**Pull request merged**